plugins/generator-1.21.1/neoforge-1.21.1/generator.yaml

```diff
@@ -85,7 +85,7 @@ base_templates:
     condition_any:
       - "${w.getGElementsOfType('biome')?filter(e -> e.spawnBiome || e.spawnInCaves || e.spawnBiomeNether || e.hasVines() || e.hasFruits())?size != 0}"
       - "${w.hasElementsOfType('feature')}"
-      - "${w.hasElementsOfType('command')}"
+      - "${w.getGElementsOfType('command')?filter(e -> !e.argsxml?contains('resource_location'))?size != 0}"

```

Comment:
Please measure the performance impact of this, it is complex condition on base files